homeassistant/components/aprilaire/entity.py

```diff
@@ -0,0 +1,46 @@
+"""Base functionality for Aprilaire entities."""
+
+from __future__ import annotations
+
+import logging
+
+from pyaprilaire.const import Attribute
+
+from homeassistant.helpers.update_coordinator import BaseCoordinatorEntity
+
+from .coordinator import AprilaireCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BaseAprilaireEntity(BaseCoordinatorEntity[AprilaireCoordinator]):
```

Comment:
```suggestion
class BaseAprilaireEntity(CoordinatorEntity[AprilaireCoordinator]):
```

Reply:
@joostlek This was based on the feedback from @emontnemery. It doesn't ultimately matter to me, though `CoordinatorEntity` adds nothing of value.

Reply:
Oh in that case ignore my comments

Reply:
To add to this, the rationale for using `BaseCoordinatorEntity` and `BaseDataUpdateCoordinatorProtocol` instead of `CoordinatorEntity` and `DataUpdateCoordinator` is that the integration does not have a coordinator based on our `DataUpdateCoordinator`.

The coordinator was in an earlier version derived from `DataUpdateCoordinator`, but did not use any of the base class' methods for data update etc., and I requested to change to `BaseDataUpdateCoordinatorProtocol` to make it clearer the coordinator's behavior is entirely cutom.